tests/rptest/tests/datalake/datalake_verifier.py

```diff
@@ -193,9 +198,13 @@ def _all_offsets_translated(self):
                     )
                     return False
 
-                if self._max_queried_offsets[p.id] < p.high_watermark - 1:
+                max_offset_to_check = p.high_watermark - 1
+                if self._max_offset_check_override:
```

Comment:
nit: `is not None`, given _max_offset_check_override = 0 also skips this check?

Reply:
removed this code.